src/show/pages/private/MyProjects/index.js

```diff
@@ -109,7 +96,10 @@ const MyProjects = () => {
               <div className='flex items-center justify-between w-full'>
                 <div className='w-full'>
                   <Button
-                    onClick={e => deleteProject(e)}
+                    onClick={e => {
+                      handleDeleteClick(e);
+                      setDeleteProjectModal(false);
```

Comment:
Close the modal in the handledeleteclick function and you can do this in one line after.

Reply:
roger that